### PR TITLE
A sample database for testing

### PR DESCRIPTION
**Pull request by prerit2010**

### CODE REVIEW

**Comment by rgaiacs** [comment]
@prerit2010 Please run

```
$ git checkout sample_db
$ git rebase master
$ git push -f origin master
```

File: manage.py [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
 migrate = Migrate(application, db)
  
 manager = Manager(application)
 manager.add_command('db', MigrateCommand)
+
+@manager.command
+def import_db():
+    try:
+        con = sqlite3.connect(SAMPLE_DATABASE_URI)
+    except:
```
Comment (rgaiacs) [inline manage.py]: Can we use a less general exception?

File: manage.py [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
+        print("\nCould not connect to database")
+        sys.exit(0)
+    cursor = con.execute("SELECT * from user_system_info limit 1")
+    result= cursor.fetchone()
+    if result:
+        choice = input("Database already contains some data. Do you wish to delete all data and overwrite? (y/n) :")
```
Comment (rgaiacs) [inline manage.py]: Instead of query the user, could you request to run the command with `--overwrite`?

File: manage.py [hunk 3]
```diff
@@ -1,12 +1,38 @@
...
+        if choice == 'y' or choice == 'Y':
+            con.execute("DELETE FROM user_system_info")
+            con.execute("DELETE FROM successful_installs")
+            con.execute("DELETE FROM failed_installs")
+            con.execute("DELETE FROM attempts")   
+        else:
+            sys.exit(0)
+    print("\nImporting sample data....\n")
+    f = open('sample.sql', 'r')
```
Comment (rgaiacs) [inline manage.py]: Why not use

```
with open() as f:
    sql = f.read()
```

here?

File: manage.py [hunk 4]
```diff
@@ -1,12 +1,38 @@
...
+        con = sqlite3.connect(SAMPLE_DATABASE_URI)
+    except:
+        print("\nCould not connect to database")
+        sys.exit(0)
+    cursor = con.execute("SELECT * from user_system_info limit 1")
+    result= cursor.fetchone()
+    if result:
+        choice = input("Database already contains some data. Do you wish to delete all data and overwrite? (y/n) :")
+        if choice == 'y' or choice == 'Y':
+            con.execute("DELETE FROM user_system_info")
```
Comment (rgaiacs) [inline manage.py]: Can we use a for-loop here?

```
for table in []:
    con.execute()
```

**Comment by prerit2010** [comment]
Does it look fine now ?

**Pull request merged**